packages/calcite-components/src/components/stepper/stepper.e2e.ts

```diff
@@ -460,8 +475,10 @@ describe("calcite-stepper", () => {
           const item2 = wrapper.shadowRoot.querySelector<HTMLElement>("#item-2");
           await new Promise<void>((resolve) => requestAnimationFrame(() => resolve()));
```

Comment:
I totally missed this earlier, but there's a test util ([`waitForAnimationFrame`](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/src/tests/utils.ts#L279)) you can use to simplify. Not critical for this PR.

Reply:
Tried this but for some reason it is not working in browser context. Am i missing anything ?

Reply:
Oh, my bad. I didn't realize this was within an `evaluate` callback. Ignore my comment then. 

There is a way to split up the `evaluate` to allow us use the helper in the test context, but we can tackle that later.